fastestimator/dataset/horse2zebra.py

```diff
@@ -12,66 +12,87 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-"""
-Download horse2zebra dataset from https://people.eecs.berkeley.edu/~taesung_park/CycleGAN/datasets/horse2zebra.zip
+"""Download horse2zebra dataset from https://people.eecs.berkeley.edu/~taesung_park/CycleGAN/datasets/horse2zebra.zip.
 """
 import os
 import zipfile
-import tempfile
 from glob import glob
+from pathlib import Path
 
 import pandas as pd
 import wget
 
+from fastestimator.util.wget import bar_custom, callback_progress
+
+wget.callback_progress = callback_progress
+
 
-def _create_csv(img, key_name, parent_path, csv_name):
+def _create_csv(img_path, key_name, parent_path, csv_path):
     df = pd.DataFrame()
-    df[key_name] = [img_filename.replace(parent_path, '.') for img_filename in img]
-    df.to_csv(csv_name, index=False)
+    df[key_name] = img_path
+    df[key_name] = df[key_name].apply(lambda x: os.path.relpath(x, parent_path))
+    df[key_name] = df[key_name].apply(os.path.normpath)
+    df.to_csv(csv_path, index=False)
+    print("Data summary is saved at {}".format(csv_path))
 
 
 def load_data(path=None):
-    """Downloads the horse2zebra dataset to local storage, if not already downloaded. This will generate 4 csv files
+    """Download the horse2zebra dataset to local storage, if not already downloaded. This will generate 4 csv files
     (trainA, trainB, testA, testB), which contain all the path information.
 
     Args:
-        path (str, optional): The path to store the horse2zebra data. Defaults to None, will save at
-        `tempfile.gettempdir()`.
+        path (str, optional): The path to store the horse2zebra data. When `path` is not provided, will save at
+            `fastestimator_data` under user's home directory.
 
     Returns:
-        string: path to trainA csv file.
-        string: path to trainB csv file.
-        string: path to testA csv file.
-        string: path to testB csv file.
-        string: path to data directory.
+        (tuple): tuple containing:
+            train_a_csv (str): Path to trainA csv file.
+            train_b_csv (str): Path to trainB csv file.
+            test_a_csv (str): Path to testA csv file.
+            test_b_csv (str): Path to testB csv file.
+            path (str): Path to data root directory.
+
     """
+    home = str(Path.home())
+
     if path is None:
-        path = os.path.join(tempfile.gettempdir(), ".fe", "HORSE2ZEBRA")
-    if not (os.path.exists(path)):
-        os.makedirs(path)
+        path = os.path.join(home, 'fastestimator_data', 'horse2zebra')
```

Comment:
should we extract all of this path management into a common utility function so that no one does it wrong in the future?

Reply:
you r right!   i will make the change over weekend